doc/source/whatsnew/v0.17.0.txt

```diff
@@ -87,9 +87,12 @@ Backwards incompatible API changes
 
 .. _whatsnew_0170.api_breaking.other:
 
-.. _whatsnew_0170.api_breaking.convert_objects:
+
 Changes to convert_objects
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. _whatsnew_0170.api_breaking.convert_objects:
```

Comment:
It didn't seem to work (there was no <a href="#api_breaking_convert_objects"> in the 0.17 doc build so I thought there might be something weird with multiple x-refs together.